anda/tools/arduino-lab-micropython-installer/arduino-lab-micropython-installer.spec

```
@@ -0,0 +1,45 @@
+%define debug_package %{nil}
+
+%electronmeta
+
+Name:           lab-micropython-installer
+Version:        1.4.0
+Release:        1%?dist
+License:        AGPL-3.0 AND %electron_license
+Summary:        This repository hosts the entire code of the Arduino MicroPython Installer tool
+URL:            https://github.com/arduino/lab-micropython-installer
+Source0:        %url/archive/refs/tags/v%version.tar.gz
+Source1:        micropython-installer.desktop
+Packager:       Owen Zimmerman <owen@fyralabs.com>
+Requires:       xdg-utils gtk3 libnotify nss libXtst xdg-utils libdrm libxcb
+BuildRequires:  anda-srpm-macros
+BuildRequires:  pnpm
+Provides:       arduino-lab-micropython-installer
+
```

It's very ugly but the macro should be under `Name:` due to the bcond for the package name (similar limitation with `%forgemeta`).

Also clean up deps the meta macro handles (`xdg-utils` was also listed twice).
```suggestion
Name:           lab-micropython-installer
%electronmeta
Version:        1.4.0
Release:        1%?dist
License:        AGPL-3.0 AND %electron_license
Summary:        This repository hosts the entire code of the Arduino MicroPython Installer tool
URL:            https://github.com/arduino/lab-micropython-installer
Source0:        %url/archive/refs/tags/v%version.tar.gz
Source1:        micropython-installer.desktop
Packager:       Owen Zimmerman <owen@fyralabs.com>
Requires:       libdrm libxcb
BuildRequires:  anda-srpm-macros
BuildRequires:  pnpm
Provides:       arduino-lab-micropython-installer

```